setup-utils/install-mxnet-ubuntu-python.sh

```diff
@@ -5,7 +5,7 @@
 ######################################################################
 set -e
 
-MXNET_HOME="$HOME/mxnet/"
+MXNET_HOME=${PWD}"/../../mxnet/"
```

Comment:
@mli @cyberfire I encountered this problem yesterday too. I launched a pull request-->"fix error: cd: /home/username/mxnet/: No such file or directory #4011", but it build failed. However, I use the command "bash install-mxnet-ubuntu-python.sh", the mxnet is installed successfully in my ubuntu 14.04 in dir "/usr/local/app/mxnet".